### DISCUSSION TITLE
Add HEIC Support

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#163] [Q&A] [answered]
HEIC, the default image format on iPhones, doesn't seem to be supported. 

I have found pyheif, [https://github.com/carsales/pyheif](https://github.com/carsales/pyheif), which can generate PIL readable bytecode. 

When a heic file is uploaded, a jpg should be generated for viewing and when people download they should get the original heic file.

What do you think?

**Comment by mgogoulos** [comment, marked as answer]
This is valid, please open an issue and I will give it the enhancement label so it can be worked at some point
